heltour/tournament/admin.py

```diff
@@ -992,8 +1043,10 @@ def team_manage_players_view(self, request, object_id):
                 purple_players.add(sp.player)
 
         expected_ratings = {sp.player: sp.expected_rating(league) for sp in season_player_objs}
+        season_started = bool(Round.objects.filter(season=season, publish_pairings=True).count())
```

Comment:
@colwem Can you change this from `bool(...count())` to `...exists()`? I'll merge after that

Reply:
I thought I got both of those

Reply:
I'm onnit